server/src/Application/Validators/Course/UpdatedCourseValidator.cs

```diff
@@ -0,0 +1,12 @@
+using FluentValidation;
+using StudyZen.Domain.Entities;
+
+namespace StudyZen.Application.Validators;
+
+public class UpdatedCourseValidator : AbstractValidator<Course>
+{
+    public UpdatedCourseValidator()
+    {
```

Comment:
Not sure why validating the entity is needed, when the service takes in DTOs

Reply:
If you want to check if it's whitespace, you can use `Trim()` method on request properties to remove trailling whitespace before passing it to service.

Reply:
Also there Fluent Validation has `When` and `Unless` rules which you probably needed